api/src/opentrons/protocol_api/instrument_context.py

```diff
@@ -904,12 +911,12 @@ def drop_tip(
 
                 * You're using a GEN2 pipette, not a GEN1 pipette.
                 * You've tested ``home_after=False`` extensively with your
-                  particular pipette and your particular tips.
+                  pipette and tips.
                 * You understand the risks described below.
 
             The ejector shroud that pops the tip off the end of the pipette is
```

Comment:
another shroud! let's send it to the grave too.
```suggestion
            The ejector that pops the tip off the end of the pipette is
```